tachyon/zk/plonk/halo2/argument.h

```diff
@@ -129,23 +129,40 @@ class Argument {
   // of instance evaluation-formed columns. (a.k.a. Batch IFFT)
   static std::vector<std::vector<Poly>> GenerateInstancePolys(
       ProverBase<PCS>* prover,
-      std::vector<std::vector<Evals>> instance_columns_vec) {
-    return base::Map(instance_columns_vec,
-                     [prover](const std::vector<Evals>& instance_columns) {
-                       return base::Map(
-                           instance_columns,
-                           [prover](const Evals& instance_column) {
-                             if constexpr (PCS::kQueryInstance) {
-                               prover->CommitAndWriteToProof(instance_column);
-                             } else {
-                               for (size_t i = 0; i < prover->pcs().N(); ++i) {
-                                 CHECK(prover->GetWriter()->WriteToTranscript(
-                                     *instance_column[i]));
-                               }
-                             }
-                             return prover->domain()->IFFT(instance_column);
-                           });
-                     });
+      const std::vector<std::vector<std::vector<F>>>& instance_3d_vec) {
```

Comment:
Is there a reason to name `instance_3d_vec`? If not, please name it `instance_columns_vec` for consistency with other places.